float32 q1_dot
float32 q2_dot
float32 q3_dot
---
float32 Vx
float32 Vy
float32 Vz
float32 Wx
float32 Wy
float32 Wz
